docs/data_engineers/accessing_data/elt_pipeline/elt_destinations/postgres.md

```diff
@@ -0,0 +1,134 @@
+---
+sidebar_position: 1
+---
+
+# Postgres Destination
+
+In order to create an end-to-end test of the ELT pipeline apart from the Source which is the KYVE network, you should
+configure a destination where the `records` will be stored.
+
+This page will guide you through setting up an ELT pipeline to fetch data from
+a [KYVE data pool](https://docs.kyve.network/basics/pools.html) and import it into a local `Postgres` database.
+
+## Postgres Local Deployment
+
+In this step, you will deploy a local `Postgres` database using Docker.
+
+1. Run the following command to create a container with the latest Postgres image:
+
+   ```bash
+   docker run --name local-psql -v local_psql_data:/var/lib/postgresql/data -p 5432:5432 -e POSTGRES_PASSWORD=pass -d postgres
+   ```
+
+2. Access the Postgres terminal to set up a database for the pipeline's destination:
+
+   ```bash
+   docker exec -it local-psql psql -U postgres
+   ```
+
+3. In the Postgres terminal, run the following commands:
+
+   ```sql
+   CREATE DATABASE kyvetest;
+   ```
+
+   ```sql
+   CREATE USER airbyte_user WITH ENCRYPTED PASSWORD 'pass';
+   ```
+
+   ```sql
+   GRANT ALL PRIVILEGES ON DATABASE kyvetest to airbyte_user;
+   ```
+
+   ```sql
+   \connect kyvetest
+   ```
+
+   ```sql
+   GRANT ALL ON SCHEMA public to airbyte_user;
+   ```
+
+   A database named `kyvetest` has been created and the user `airbyte_user` with password `pass` now
+   has read and write access.
+
+4. Install the [pgAdmin 4](https://www.pgadmin.org/download/) management tool.
+
+5. Click on "Add New Server" and create a connection with the `kyvetest` database.
+   As show in the figure bellow you should add the following parameters:
+
+    - **Maintenance database**: `kyvetest`
+    - **Username**: `airbyte_user`
+    - **Password**: `pass` (in case you did not modify it from step 3.1)
+
+   <br></br>
+
+   <img src="/img/elt/pgAdmin4_connection.jpg"/>
+
+## Create an ELT pipeline
```

Comment:
if we want parity with the Snowflake docs:
```suggestion
## Configure Airbyte
```

Reply:
LGTM.